openio-sds/openio-sds.spec

```diff
@@ -140,10 +140,6 @@ Requires:       python-gunicorn    >= 19.4.5
 Requires:       python-flask,python-eventlet,python-zmq,python-redis,python-requests >= 2.6.0,python-plyvel,PyYAML,python-futures
 Requires:       pyxattr            >= 0.4
 Requires:       python-simplejson  >= 2.0.9
-# Python oiopy dependencies
-Requires:       python-eventlet >= 0.15.2, python-requests >= 2.6.0, python-cliff-tablib, python-cliff >= 1.13, python-tablib, python-pyeclib >= 1.2.0
-Requires:       python-urllib3 >= 1.12
-Provides:       python-oiopy
```

Comment:
This `Provides` is harmless, and is there to ease dependency resolution.
If someone has explicitly installed `python-oiopy` and:
- no `python-oiopy` package is available;
- no other package provides `python-oiopy`;

the package installer will fail during dependency resolution.